internal/kafka/internal/workers/kafka_mgrs/accepted_kafkas_mgr.go

```diff
@@ -152,6 +154,7 @@ func (k *AcceptedKafkaManager) markTheAssignedKafkaAsFailedOrAllowRetryStrimziVe
 		return errors.Wrapf(err, "failed to update failed kafka %s", kafka.ID)
 	}
 	metrics.UpdateKafkaRequestsStatusSinceCreatedMetric(constants2.KafkaRequestStatusFailed, kafka.ID, kafka.ClusterID, durationSinceCreation)
+	metrics.UpdateKafkaRequestsStatusSinceLastSetMetric(constants2.KafkaRequestStatusFailed, kafka.ID, kafka.ClusterID)
```

Comment:
Same as above, let's only update this if the status is not already `Failed`. This is called on every reconcile so it will get updated every few seconds.